app/client/lib/ethereum/observeCustomContracts.js

```diff
@@ -76,29 +76,39 @@ observeCustomContracts = function(){
         @method added
         */
         added: function(newDocument) {
+            // Disables it by default
+            CustomContracts.update(newDocument._id, {$set: {
+                disabled: true
+            }});
 
+            // check for logs
+            addLogWatching(newDocument);
```

Comment:
If it's disabled by default why do you add a watch log? Shouldn't this be moved after line `92`?